deploy.sh

```diff
@@ -4,9 +4,9 @@ pip install -r requirements.txt
 mkdir deploy/
 
 if [ -d "venv/lib/python3.6" ]; then
-    cp -r venv/lib/python3.6/site-packages/ deploy/
+    cp -r venv/lib/python3.6/site-packages/. deploy/
```

Comment:
we should try to extend this to any python virtual enviornment

Reply:
cffi has the py version in it's name, so it'd actually die on any other python that lambda can't run... I'm thinking of changing it to an error message in the script otherwise.